models/Latest.js

```diff
@@ -0,0 +1,15 @@
+var keystone = require('@twreporter/keystone');
+var transform = require('model-transform');
+
+var Latest = new keystone.List('Latest', {
+	autokey: { from: 'name', path: 'key', unique: true },
+});
+
+Latest.add({
+	name: { label: '最新', type: String, required: true },
+	// TODO: save editing history for viewing in keystone UI
+	history: { label: '編輯紀錄', type: String, noedit: true },
```

Comment:
In this phase, we would not implement `history` feature, could we leave the todo comment and delete line 11 for now ?